eventmesh-trace-plugin/eventmesh-trace-zipkin/src/main/java/org/apache/eventmesh/trace/zipkin/ZipkinTraceService.java

```diff
@@ -144,10 +143,9 @@ public String get(Map<String, Object> carrier, String key) {
 
     @Override
     public void inject(Context context, Map<String, Object> map) {
-        textMapPropagator.inject(context, map, new TextMapSetter<Map<String, Object>>() {
-            @Override
-            public void set(@Nullable Map<String, Object> carrier, String key, String value) {
-                map.put(key, value);
+        textMapPropagator.inject(context, map, (cr, key, value) -> {
```

Comment:
An issue has been raised here, and a pr has been submitted.issue #3480

Reply:
Thanks for your review. That pr only replace with lambda, but  it seems that the variable `map` in lambda also should be modified.
https://github.com/apache/eventmesh/blob/124ec9f17897070afff85524596b594cec6d3bc1/eventmesh-trace-plugin/eventmesh-trace-zipkin/src/main/java/org/apache/eventmesh/trace/zipkin/ZipkinTraceService.java#L147

Reply:
You can comment on the changes under that pr. You are very welcome to discuss it together.

Reply:
OK. I will revoke this part of the modification.